### DISCUSSION TITLE
How do I select formatter?

### DISCUSSION THREAD

**Discussion by sevenc-nanashi** [neoclide/coc.nvim#4983] [Q&A] [answered]
I have two formatters: syntax_tree via [coc-ruby-syntax-tree](https://github.com/yaegassy/coc-ruby-syntax-tree) and RuboCop via `languageserver` setting.
I'd like to use syntax_tree for formatting, but it look like coc tries to use rubocop one.
How do I make coc to use syntax_tree for formatting?

**Comment by fannheyward** [comment, marked as answer]
Add `"disabledFeatures": ["formatting"]` to your RuboCop settings to disable RuboCop's formatting.
